.cirrus.yml

```diff
@@ -228,6 +229,17 @@ debian12_binary_task:
   env:
     ZEEK_CI_CONFIGURE_FLAGS: *BINARY_CONFIG
 
+debian12_spicy_ssl_task:
+  container:
```

Comment:
Further down is a `ubuntu22_spicy_task`. It would be sufficient to add `--enable-spicy-ssl` there I believe? The nice side-effect is that the task also submits to zeek-benchmarker, allowing some automatic recording of performance differences between the BinPAC and Spicy versions.

Reply:
That sounds good :)